mpl_data_cast/gui/preferences.py

```diff
@@ -50,6 +56,9 @@ def reload(self):
                 widget.setText(str(value))
             elif isinstance(widget, QtWidgets.QSpinBox):
                 widget.setValue(int(value))
+            elif isinstance(widget, QtWidgets.QComboBox):
```

Comment:
Please change to `elif widget is self.comboBox_recipe:`, because there might be more comboboxes in the future and simply checking for the class is too general.